src/main/java/core/basesyntax/service/impl/MyFileReaderImpl.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReader;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MyFileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String filePath) {
+        List<String> lines = new ArrayList<>();
+        InputStream inputStream = getClass().getClassLoader().getResourceAsStream(filePath);
+        if (inputStream == null) {
```

Comment:
Instead of throwing a generic RuntimeException, it would be better to throw a more specific exception, such as `FileNotFoundException` to indicate that the file was not found. This makes error handling more precise and informative for the caller.